examples/index.json

```diff
@@ -195,6 +195,27 @@
                 "verifyOutput": true,
                 "isLearnByExample": true
             },
+            {
+                "name": "Tuples",
+                "url": "tuples",
+                "verifyBuild": true,
+                "verifyOutput": true,
+                "isLearnByExample": true
+            },
+            {
+                "name": "Destructure tuples",
+                "url": "destructure-tuples",
+                "verifyBuild": true,
+                "verifyOutput": true,
+                "isLearnByExample": true
+            },
+            {
+                "name": "Rest type in tuples",
+                "url": "rest-type-in-tuples",
+                "verifyBuild": true,
+                "verifyOutput": true,
+                "isLearnByExample": true
+            },
```

Comment:
We can probably move this to a new section named "Lists" similar to what we have done for mappings.

Reply:
What is the Most suitable name, "Lists" or "Tuples"?

Reply:
Meant to highlight L192-L218, so it should be "Lists".

Reply:
+1. Will Update.

Reply:
Updated in 23f694ee077071f2aff851d74783a0d0e30fb231.